README.md

```diff
@@ -1,9 +1,51 @@
-# [aleph-node](https://alephzero.org/)
+[![LOGO][aleph-logo]][aleph-homepage]
 
-[![unit-tests](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/unit_tests.yml/badge.svg)](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/unit_tests.yml)
+[![Unit tests][unit-tests-badge]][unit-tests]
+[![E2E Tests][e2e-tests-badge]][e2e-tests]
+[![Apache 2.0 Licensed][license-image]][license-link]
 
-[![integration-tests](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/integration_tests.yml/badge.svg)](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/integration_tests.yml)
 
-[![deploy](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/deploy.yml/badge.svg)](https://github.com/Cardinal-Cryptography/aleph-node/actions/workflows/deploy.yml)
+This repository contains the Rust implementation of [Aleph Zero][aleph-homepage] blockchain node based on the [Substrate][substrate-homepage] framework.
 
-Node implementation for aleph blockchain built with Substrate framework.
+Aleph Zero is an open-source layer 1 blockchain platform focused on privacy, scalability and energy efficiency. It is based on a unique, peer-reviewed consensus algorithm - AlephBFT (as described in our [paper][aleph-bft-paper] and implemented [here][aleph-bft-link]).
+
+Aleph node is based on a Substrate node where the default finality gadget (GRANDPA) has been replaced with AlephBFT. Block authoring is realized with Substrate's Aura. The chain is run in periodic sesssions (900 blocks each) utilizing the Session pallet. The authorities in each session serve for both Aura and AlephBFT authorities, and on top of that are responsible for running the Aggregator protocol producing multisignatures of finalized blocks.
+
+
+### Installation
+
+1. Set up the Rust development environment following the instructions [here][rust-installation]
+2. Install the Substrate framework as described [here][substrate-installation]
+3. Clone this repo to your machine
+4. Run `cargo build --release -p aleph-node`
+
+### Running
+
+To experiment with Aleph Node you can locally run a small blockchain network using the `run_nodes.sh` script from the root of this repository.  Please consult the script or the output of `run_nodes.sh -help` for additional parameters (like the number of nodes etc.). The script starts multiple instances of Aleph Node on your local machine, so please adjust the number of nodes carefully with performance of your system in mind. By default 4 nodes are started.
```

Comment:
this is one option, but one can also run a small chain using the docker compose scripts: 

```bash
.github/scripts/run_consensus. 
```
Maybe worth it to document both?

Reply:
Didn't know about that ^^' I'll try it out and add a mention here

Reply:
After trying it for some time I run into some compatibility problems - the instructions would need to be somehow OS distro dependent and I think it would be a bit of an overkill to include so much details for an alternative running way.